datasets/pd12m.yaml

```diff
@@ -0,0 +1,41 @@
+Name: PD12M
+Description: PD12M is a collection of 12.4 million CC0/PD image-caption pairs for the purpose of training generative image models.
+Documentation: https://huggingface.co/datasets/Spawning/PD12M
+Contact: info@spawning.ai
+ManagedBy: Spawning
+UpdateFrequency: Data will be adjusted as infringing works are discovered, improved provenance is acquired, or infringing captions are discovered.
+Tags:
+- image processing
+- machine learning
+- media
+- art
+- deep learning
+- labeled
+License: https://cdla.dev/permissive-2-0/
+Resources:
+  - Description: Image files
+    ARN: arn:aws:s3:::pd12m
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: Working with the Metadata
+      URL: https://huggingface.co/datasets/Spawning/PD12M
+      AuthorName: Spawning
+    - Title: Downloading Images
+      URL: https://huggingface.co/datasets/Spawning/PD12M/blob/main/tutorials/images.md
+      AuthorName: Spawning
+  Tools & Applications:
+    - Title: Hugging Face Dataset
+      URL: https://huggingface.co/datasets/Spawning/PD12M
+      AuthorName: Spawning
+    - Title: Source.Plus
+      URL: https://source.plus/collections/pd12m-mxenifxs
```

Comment:
@Padge91 this URL seems to be 404'ing for me. Can you double check this is correct?

Reply:
@kc00k Apologies, this has been fixed in a new commit.

Reply:
@Padge91 Thank you. Your branch is now out of date, once you merge main back into your branch I will move forward with merging your branch in.

Reply:
@kc00k Done!